subsys/bluetooth/audio/Kconfig.baps

```diff
@@ -93,23 +93,23 @@ config BT_AUDIO_UNICAST_CLIENT_GROUP_STREAM_COUNT
 config BT_AUDIO_UNICAST_CLIENT_PAC_COUNT
 	int "Basic Audio Profile PAC count"
 	default 2
-	range 0 4
+	range 0 146
```

Comment:
`The limit of 73 * 2 = 146 is based on the maximum size of an
GATT attribute (512), and the minimum size of a PAC record (7)
makes it a total of 73 PAC records for each direction.`

I guess the assumption here is incorrect, as you take into account `minimum size of a PAC record`. If PAC record will larger, the PAC will exceed the 512 bytes anyway. So there is no need to set the limit (based on such calculations) here as you cannot ensure the PAC list will not exceed it (so the Long Read operation should be handled).

Reply:
I am not sure I follow. 

> So there is no need to set the limit (based on such calculations) here as you cannot ensure the PAC list will not exceed it 

The absolute maximum number of PAC records a device can hold, based on the ATT restriction for maximum attribute sizes, is 73. 

> so the Long Read operation should be handled

Whatever operation used to read the PAC records doesn't change the fact that 512 is the maximum ATT attribute size (specified by the core spec). Your Sink or Source PAC characteristic cannot "legally" be larger than 512, and must thus at maximum contain 73 records.

Reply:
![image](https://user-images.githubusercontent.com/2520413/192753204-928950f4-eee5-449d-aa40-03957748c35a.png)

Reply:
OK, thanks for the Spec quote, I have to say I forgot about 512 bytes limitation. But as I wrote, the you assume the PAC record is 7 bytes, but that's the minimum size. If you'll have a 6 PAC records 100 bytes each you'll exceed 512 bytes anyway, even if 6 < 146

Reply:
> Keep in mind, our current implementation supports 1 Sink PAC and 1 Source PAC characteristics

PACS can only have 1 of each, but they may hold multiple records:
![image](https://user-images.githubusercontent.com/2520413/193763142-3a9826ad-f6fd-450a-977c-f44b2bd1cffa.png)

And yes, of course 6 PAC records may 512 octets, but that doesn't change the theoretical maximum number of records. 

What is that you'd to change here? 73 PAC records for each direction is the theoretical maximum (total of 146), but is it because you'd like the maximum number of records support to set to a lower value, e.g. 100? 20? 

At UPF at least one device had a total of 46 PAC records in a single characteristic.